### PR TITLE
Add correct data copyright footer text (NISRA)

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds correct copyright text to NISRA census schemas as described on [Trello card](https://trello.com/c/Hc5S2ltr).

### How to review 
Run NISRA census themed schemas and check if text of footer data copyright is correct.

### Checklist

* [x] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Incorrect address data copyright in footer on NI schemas](https://trello.com/c/Hc5S2ltr/4496-incorrect-address-data-copyright-in-footer-on-ni-schemas)

**Review by LJBabbage — APPROVED** [review]
I assume this got approved from whoever was going to approve it before it went into dev? Not a fan of the formatting, but it works

**Review by MebinAbraham — APPROVED** [review]

File: app/helpers/template_helpers.py [hunk 1]
```diff
@@ -50,8 +50,10 @@ def get_footer_context(language_code, static_content_urls, sign_out_url, theme):
         "newTabWarning": lazy_gettext("The following links open in a new tab"),
         "copyrightDeclaration": {
             "copyright": lazy_gettext(
-                "Crown copyright and database rights 2020 OS 100019153."
-            ),
+                "Crown copyright and database rights 2021 NIMA MOU577.501."
+            )
+            if theme == "census-nisra"
+            else lazy_gettext("Crown copyright and database rights 2020 OS 100019153."),
```
Comment (JamesGardiner) [inline app/helpers/template_helpers.py:56]: Are we 100% sure this shouldn't be 2021?

Comment (petechd) [reply app/helpers/template_helpers.py:56]: I'll check with @donnap573

Comment (petechd) [reply app/helpers/template_helpers.py:56]: Spoke to @donnap573 and in original email definitely says to put 2020

Comment (JamesGardiner) [reply app/helpers/template_helpers.py:56]: 🤷🏻‍♂️

**Pull request merged**